packages/backend/src/lib/xpHelpers.ts

```diff
@@ -0,0 +1,25 @@
+export const getNumWeeksInSeason = (): number => {
```

Comment:
```suggestion
export const getNumWeeksInSeason = (currentDate = new Date()): number => {
```

For flexibility's sake you could take in the date to compute against as an argument and give it a default value.

Reply:
Good idea